ui.content/src/main/content/jcr_root/conf/acm/settings/snippet/available/core/argument/multi_select.yml

```diff
@@ -10,6 +10,6 @@ documentation: |
   
   For example:
   ```groovy
-  args.multiSelect("foo") { label = "Foo"; options = ["bar": "Bar", "baz": "Baz"]; value = ["bar"] }
+  args.multiSelect("foo") { label = "Foo"; options = ["bar": "Bar", "baz": "Baz"]; value = ["Bar"] }
```

Comment:
the intention was to have map entries "label" => "value"; not vice versa

Reply:
yeah, but this snippet is wrong, because it provides value which is not displayed, so by mistake you have hidden value which is passed as argument but not visible in your form